components/x-follow-button/src/FollowButton.jsx

```diff
@@ -0,0 +1,78 @@
+import { h } from '@financial-times/x-engine';
+import classNames from 'classnames';
+import styles from './styles/main.scss';
+
+export const FollowButton = (props) => {
+	const {
+		useConceptNameAsButtonText = false,
```

Comment:
I know I named this, but thinking about it a bit more, we could shorten it to `conceptNameAsButtonText`?

Reply:
👍